Apps/WebClient/src/ClientApp/src/models/registerTestKitRequest.ts

```diff
@@ -6,5 +6,11 @@ export default interface RegisterTestKitRequest {
     testTakenMinutesAgo?: number;
 
     // ID of the test kit
-    testKitId?: string;
+    testKitCid?: string;
```

Comment:
Why the field name was change to testKidCid?

Reply:
That is what it is at PHSA.